src/tests/runtime_bignum_test.nr

```diff
@@ -682,3 +683,64 @@ fn test_sqrt_BN() {
 
     assert(sqrt_x * sqrt_x == x);
 }
+
```

Comment:
Maybe add a comment before this test to justify its existence, in case we forget why it's there. E.g. "Regression test for bug introduced in v0.3.7. See https://github.com/noir-lang/noir-bignum/pull/51."

Reply:
Added the comment to explain what the test is for